pkgs/development/tools/cask/default.nix

```diff
@@ -19,6 +19,7 @@ stdenv.mkDerivation rec {
 
   buildPhase = ''
     emacs --batch -L . -f batch-byte-compile cask.el cask-cli.el
+    patchShebangs bin/cask
```

Comment:
```suggestion
    patchShebangs --host bin/cask
```

Reply:
Thank you

Reply:
The suggested solution does not produce the expected result. As soon as I add the `--host` argument to the patchShebangs command the resulting executable is not correctly patched anymore.

Reply:
🤔 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/patch-shebangs.sh#L21

You may need to add the package to buildInput that are added to the shebang.

Reply:
@SuperSandro2000 Thanks for the hint. Adding bash to the build inputs totally solved the problem.  Instead of calling patchShebangs manually I added pre and post phase hooks to buildPhase and installPhase. This seems to be the preferred way of doing this kind of stuff in `nixpkgs`.